.deploy/cf-runtime/templates/volume-provisioner/service-account.dind-volume-provisioner.vp.yaml

```diff
@@ -3,9 +3,7 @@ kind: ServiceAccount
 metadata:
   name: {{ include "cf-vp.fullname" . }}
   labels: {{- include "cf-vp.provisionerLabels" . | nindent 4 }}
+  {{- with .Values.volumeProvisioner.serviceAccount.annotations }}
   annotations:
-{{- if .Values.volumeProvisioner.serviceAccount }}
```

Comment:
Removed this line because this condition is always true if `Values.volumeProvisioner.serviceAccount.annotations` exists